doc/lua_api.md

```diff
@@ -9381,6 +9381,13 @@ Used by `minetest.register_node`.
     -- (see `liquid_move_physics`), the movement speed will also be
     -- affected by the `movement_liquid_*` settings.
 
+    climb_factor = 1.0,
+    -- The speed at which a climbable node can be climbed is multiplied
+    -- with this number. Must not be negative. No effect if node isn't
+    -- climbable.
+    -- Note: To set the base climbing speed in your game,
+    -- change the setting "movement_speed_climb".
```

Comment:
Isn't it a bad idea to advise modders to change settings when they have no proper way to reset them? Why not advise them to use the physics override instead?

(That said, movement probably shouldn't be controlled by such settings...)